cosmos-common/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 mod pb;
+mod testing;
```

Comment:
Need to be excluded when not in test
```suggestion
#[cfg(test)]
mod testing;
```